content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
```

Comment:
```suggestion
canonicalUrl: https://gateway-api.sigs.k8s.io/blog/2022/introducing-v1beta1/
---
```
(that is to follow the pattern from https://gateway-api.sigs.k8s.io/blog/2021/introducing-v1alpha2/ - and I therefore also suggest changing the URL of the draft article on the Gateway API site)